lib/src/values/logic_value.dart

```diff
@@ -725,6 +725,14 @@ abstract class LogicValue {
   /// [eq] will return `x`.
   LogicValue eq(dynamic other) => _doCompare(other, (a, b) => a == b);
 
+  /// Not equal-to operation.
+  ///
+  /// This is different from != operator because it returns a [LogicValue] instead
+  /// of a [bool].  It does a logical comparison of the two values, rather
+  /// than exact inequality.  For example, if one of the two values is invalid,
+  /// [neq] will return `x`.
+  LogicValue neq(dynamic other) => _doCompare(other, (a, b) => a != b);
```

Comment:
The `neq` tests for `Logic` look great, but can we add some for `LogicValue` as well?  There are some tests in `logic_value_test.dart` that you could use for reference.